lib/reporter.js

```diff
@@ -84,7 +84,8 @@ class CucumberReporter {
         case Cucumber.Status.PENDING:
         case Cucumber.Status.SKIPPED:
         case Cucumber.Status.AMBIGUOUS:
-            e = 'pending'
+            e = 'skipped'
```

Comment:
It is a valid point, but then here is a different problem.
Mocha framework doesn't fire `test:start` event for pending tests, but Cucumber does.
Allure is the only reporter that suffers from it just because it is the only reporter from the standard set, that uses this event, but there is a general problem in inconsistency, that I am trying to mitigate.